src/main/java/org/prebid/server/bidder/conversant/ConversantBidder.java

```diff
@@ -90,77 +83,51 @@ public ConversantBidder(String endpointUrl, JacksonMapper mapper) {
                         .headers(HttpUtil.headers())
                         .payload(outgoingRequest)
                         .build()),
-                errors);
+                Collections.emptyList());
     }
 
-    private BidRequest createBidRequest(BidRequest bidRequest, List<BidderError> errors) {
+    private BidRequest createOutgoingRequest(BidRequest bidRequest) {
+        final BidRequest.BidRequestBuilder requestBuilder = bidRequest.toBuilder();
         final List<Imp> modifiedImps = new ArrayList<>();
-        Integer extMobile = null;
-        String extSiteId = null;
 
-        for (Imp imp : bidRequest.getImp()) {
+        for (int i = 0; i < bidRequest.getImp().size(); i++) {
+            final Imp imp = bidRequest.getImp().get(i);
+            final ExtImpConversant impExt;
             try {
-                validateImp(imp);
-                final ExtImpConversant impExt = parseImpExt(imp);
-                modifiedImps.add(modifyImp(imp, impExt));
-                if (StringUtils.isNotEmpty(impExt.getSiteId())) {
-                    extSiteId = impExt.getSiteId();
-                }
-                if (impExt.getMobile() != null) {
-                    extMobile = impExt.getMobile();
-                }
+                impExt = parseImpExt(imp);
```

Comment:
We can pass `i` as argument to `parseImpExt(..)` so this try-catch will be not needed.